### PR TITLE
Roadmap to DL added

### PR DESCRIPTION
**Pull request by take2rohit**
Please review this. 
If small changes are there modify and merge it, please.

### CODE REVIEW

File: deep-learning/README.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ In these courses, you will learn the foundations of Deep Learning, understand ho
 2. [CS231n: Convolutional Neural Networks for Visual Recognition](http://cs231n.stanford.edu/)
 
 ### Roadmap to Deep Learning with Images
```
Comment (SharathRaparthy) [inline deep-learning/README.md]: Deep learning with images sounds weird. Instead, write something like "Roadmap to deep learning for perception/vision"

File: deep-learning/README.md [hunk 2]
```diff
@@ -19,15 +19,15 @@ In these courses, you will learn the foundations of Deep Learning, understand ho
...
-3. It is assumed that you know python. If not learn it and then come back.
-4. Then you are finally ready to get started with [Deep Learning Specialization by Andrew Ng](https://www.coursera.org/specializations/deep-learning). The foundation built through these course are very essential. Make proper notes of whatever you read. 
-5. Code your artificial neural networks and Convolution Neural network from scratch using numpy.
-6. You should now pickup a framework to work with. Most members of IvLabs prefer PyTorch. 
-7. You can skip *Sequence Modelling course of Deep Learning Specialization* and do it later if you have interest. 
-8. Keep yourself updated with latest Reasearch Papers. Reddit and Twitter is highly recommended.
-9. Ask for support from seniors who have already worked on these fields.  
+1. Get started with [Introduction to Computer Vision](https://www.udacity.com/course/introduction-to-computer-vision--ud810) to get an idea of how Image Processing and Computer Vision works. You can do this course upto *Lesson 3: Linearity and Convolution* before moving to DL. This builds a strong mathematical foundation and helps you to easily grasp DL later.
```
Comment (SharathRaparthy) [inline deep-learning/README.md]: Maybe you can improve the sentence formation. Write something like this "Before jumping to learning-based techniques for vision; it is recommended to go through some of the fundamental concepts of computer vision. From our experience, we found [Introduction to Computer Vision](https://www.udacity.com/course/introduction-to-computer-vision--ud810) is a good starting point, and it covers some concepts like (name two or three concepts).

File: deep-learning/README.md [hunk 3]
```diff
@@ -19,15 +19,15 @@ In these courses, you will learn the foundations of Deep Learning, understand ho
...
+2. Once you get the hang of Computer Vision, you can start revising basic Linear Algebra alongside. You should atleast have knowledge of maths taught till 1st year of engineering (concepts like differential calculus). Keep on reading about Linear Algebra throughout the journey of DL.
```
Comment (SharathRaparthy) [inline deep-learning/README.md]: This sentence is more or less like ordering. Write something along these lines.
Once you get a hang on computer vision concepts, it is advisable to go through some of the linear algebra concepts as well. Some of them include eigendecomposition, single value decomposition, matrix and vector norms (and other imp concepts. Refer Ian Goodfellow's textbook for more references). 

It's always good to be more specific and being abstract (like linear alebra)

File: deep-learning/README.md [hunk 4]
```diff
@@ -19,15 +19,15 @@ In these courses, you will learn the foundations of Deep Learning, understand ho
...
+3. It is assumed that you know Python. If not, then go through some tutorials like [this one by Corey Schafer](https://www.youtube.com/watch?v=YYXdXT2l-Gg&list=PL-osiE80TeTt2d9bfVyTiXJA-UTHn6WwU).
```
Comment (SharathRaparthy) [inline deep-learning/README.md]: Nah! CHange this sentence.

**Review by SharathRaparthy — CHANGES_REQUESTED** [review]
Please make changes

File: deep-learning/README.md [hunk 5]
```diff
@@ -18,6 +18,16 @@ In these courses, you will learn the foundations of Deep Learning, understand ho
 1. [Introduction to Computer Vision](https://www.udacity.com/course/introduction-to-computer-vision--ud810)
 2. [CS231n: Convolutional Neural Networks for Visual Recognition](http://cs231n.stanford.edu/)
 
+### Roadmap to Deep Learning with Images
```
Comment (kartikpaigwar) [inline deep-learning/README.md]: You can say Deep Learning for Visual Recognition. Perception is a bigger category. Prerequisites will change accordingly.

File: deep-learning/README.md [hunk 6]
```diff
@@ -18,6 +18,16 @@ In these courses, you will learn the foundations of Deep Learning, understand ho
...
+1. Get started with [Introduction to Computer Vision](https://www.udacity.com/course/introduction-to-computer-vision--ud810) to get an idea of how Image Processing and Computer Vision works. You can do this course upto *Lesson 3: Linearity and Convolution* before moving to DL. This builds a strong mathematical foundation and helps you to easily grasp DL later.
```
Comment (kartikpaigwar) [inline deep-learning/README.md]: See, if one's focus is to use deep learning for object detection, segmentation (feature level understanding) etc, then understanding only Image Processing is sufficient.  [Digital image processing by Rafael C. Gonzalez](http://web.ipac.caltech.edu/staff/fmasci/home/astro_refs/Digital_Image_Processing_2ndEd.pdf) is a standard book to refer. If one's focusing on deep learning for Computer Vision/Perception application then Image Processing and complete course on Introduction to Computer Vision is requisite. Perception includes concepts of 3D perspective, Stereo, Optical flow, object tracking, visual recognition etc which are all very important. In simple words, Image Processing is kinematics and Computer Vision is dynamics.

**Review by take2rohit — COMMENTED** [review]
Changes are done according to @SharathRaparthy and @kartikpaigwar
Feel free to reopen if there are any errors

**Pull request merged**